src/test/kotlin/com/igorwojda/integer/countdown/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countDown(n: Int): List<Int> {
-    TODO("not implemented")
+    return (0..n).toList().sortedDescending()
```

Comment:
You've used downTo in other exercises. It's better to just create a range once instead of have it reversed afterwards.